docs/kit/operation-view/installation_guide.md

```diff
@@ -0,0 +1,116 @@
+Assuming the former case, the JDBC URL can be constructed as follows: `jdbc:postgresql://${NAME}:5432/$DATABASE`.
+
+Vault has a "dev mode" that is perfect for this use case.
+First, add the HashiCorp Helm repository:
+  ```bash
+  helm repo add hashicorp https://helm.releases.hashicorp.com
+  ```
+Then, as with PostgreSQL, decide on a name for your Vault instance.
+You can then roll it out using:
+  ```bash
+  helm upgrade --install $NAME hashicorp/vault --set server.dev.enabled=true --set injector.enabled=false
+  ```
+Again, similar to PostgreSQL, the Vault URL is `http://${NAME}:8200` if `$NAME` contains the string `vault`, and `http://${NAME}-vault:8200` if not.
+The token to use in dev mode is simply the string `root`.
+For reasons of simplicity, we disable the injector, as making use of it would require further configuration of the Vault instance.
+
+The current version of the chart requires a static Vault token that needs to be valid for the lifetime of the deployment.
+In a more advanced deployment, Vault could be configured with the Kubernetes auth method and a role granting access to the relevant secrets to the deployment's service account.
+Assuming the injector is enabled in this case, the deployment could be annotated as follows:
+
+- `vault.hashicorp.com/agent-cache-enabled`: `true`
+- `vault.hashicorp.com/agent-cache-use-auto-auth-token`: `force`
+
+The deployment should then be modified to expect the Vault API at `http://localhost:8200`.
+The token should be set to a garbage value, as the agent will overwrite it with the real token.
+
+Setup
+------------
+1. Following mandatory properties will be required to fully configure and run the Connector in step 4 for use with Hashicorp Vault and PostgreSQL:
+    - `daps.url`: the URL of the DAPS service (e.g. https://daps1.int.demo.catena-x.net)
+    - `daps.clientId`: the client ID of the Connector (syntax: SKI:AKI)
+    - `vault.hashicorp.enabled` to `true`
+    - `vault.hashicorp.url`: the URL of the Hashicorp Vault service
+    - `vault.hashicorp.token`: the token of the Hashicorp Vault service to access the secrets
+    - `postgresql.enabled` to `true`
+    - `postgresql.jdbcUrl`: JDBC URL of the PostgreSQL database
+    - `postgresql.username`: username of the PostgreSQL database
+    - `postgresql.password`: password of the PostgreSQL database
+    - `controlplane.endpoints.authKey`: auth key of the control plane
+    - `backendService.httpProxyTokenReceiverUrl`: URL of the HTTP proxy token receiver service
+2. Further Connector specific properties can be configured:
+    - `internationalDataSpaces.id`: the id of the IDS connector
+    - `internationalDataSpaces.description`: the description of the IDS connector
+    - `internationalDataSpaces.title`: the title of the IDS connector
+    - `internationalDataSpaces.maintainer`: the maintainer of the IDS connector
+    - `internationalDataSpaces.curator`: the curator profile of the IDS connector
```

Comment:
Are these "internationalDataspaces" configuration values still used because they have no corresponding item in the DSP specifications?  These should probably be removed (and "InternationalDataSpaces" no longer used)